tests/unit_tests/test_wave_06.py

```diff
@@ -109,17 +110,28 @@ def test_swap_best_by_category():
         my_priority="Clothing",
         their_priority="Decor"
     )
-
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
     # Assertions should check:
     # - That the results is truthy
     # - That tai and jesse's inventories are the correct length
     # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
+    assert result
```

Comment:
Since `swap_best_by_category` returns `True` we could also do `assert result is False` here as well. Though, I see the note to just test for truthyness.